api/v1beta1/dynakube_conversion.go

```diff
@@ -183,8 +183,6 @@ func (dst *DynaKube) ConvertFrom(srcRaw conversion.Hub) error {
 }
 
 func convertToDeprecatedActiveGateCapability(dst *CapabilityProperties, src *v1alpha1.CapabilityProperties) {
-	dst.Enabled = true
```

Comment:
Is this not needed?

Reply:
Its not part of CapabilityProperties, moved it out before the func call